deployment/sriov-network-operator/values.yaml

```diff
@@ -27,6 +27,9 @@ operator:
   resourcePrefix: "openshift.io"
   cniBinPath: "/opt/cni/bin"
   clusterType: "kubernetes"
+  metricsExporter:
+    port: "9110"
+    secretName: "metrics-exporter-cert"
```

Comment:
can we do the same layout as in admissionControllers ?

i.e

```
operator:
  metricsExporter:
    port: "9110"
    certificates:
      secretName: "metrics-exporter-cert"
```

Reply:
sure!